test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/PackageSigningUtilitiesTests.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace NuGet.Packaging.Test
 {
-    public class SigningUtilityTests
+    public class PackageSigningUtilitiesTests
```

Comment:
For unit tests, the class name is `<TypeUnderTest>Tests`.  Test names should be `<MemberUnderTest>_Condition_ExpectedResult`.  That's what you started with here.

There should be 2 separate files:

* CertificateUtilityTests.cs with a test class name of `CertificateUtilityTests`.  This file should only contain 
`CertificateUtility` tests.
* CertificateChainUtilityTests.cs with a test class name of `CertificateChainUtilityTests`.  This file should only contain `CertificateChainUtility` tests.